account_balance_line/README.rst

```diff
@@ -6,22 +6,23 @@
 Account Balance Line
 ====================
 
-Add a balance total for grouped lines in move line view.
+This module adds a balance total for grouped lines in move line view.
 
 Balance field will only be shown when move lines are grouped by account
 or filtered by account.
 
 Usage
 =====
 
+If you're an accounting adviser, enable developer mode and go to
```

Comment:
Put the markers I have told you in the comment: `#.`. They serve for making an unordered bullet list in RST (like `<ul>` in HTML)